src/components/PopoverWithoutOverlay/index.js

```diff
@@ -11,6 +11,8 @@ import * as Modal from '@userActions/Modal';
 
 function Popover(props) {
     const {onOpen, close} = React.useContext(PopoverContext);
+    const removeOnClose = useRef(() => {});
```

Comment:
Is this reference to removeOnClose needed here? Could we move the declaration to `useEffect` in line 29 and call this function in return for the clear function?